src/components/ArmorModal.js

```diff
@@ -0,0 +1,357 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import {
+  Card, withStyles, CardHeader, CardContent,
+  Table, TableHead, TableRow, TableBody, TableCell,
+  Tooltip, CardMedia, CardActions, Button, Typography,
+  FormControl, Select, MenuItem, InputLabel, OutlinedInput,
+  IconButton,
+} from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+import { Formik } from "formik";
+import {
+  Radar, RadarChart, PolarGrid, PolarAngleAxis, PolarRadiusAxis,
+} from "recharts";
+import _ from "lodash";
+import { armorToLoadout } from "store/ducks/Loadouts";
+import { SlidingModal, Panel } from "components/layout";
+import { StyledRecharts } from "screens/Styles";
+
+const styles = () => ({
+  imagePanel: {
+    width: "100px",
+    height: "100px",
+    margin: "auto",
+  },
+  betterTooltip: {
+    fontSize: "16px",
+    backgroundColor: "black",
+  },
+  select: {
+    top: "626px",
+    background: "red",
+  },
+  newTitle: {
+    color: "hsl(205,11%,80%)",
+    fontWeight: 600,
+  },
+});
+
+const panelBtn = {
+  cursor: "pointer",
+  padding: "8px",
+};
+
+class ArmorModal extends Component {
+  state = {
+    swapImage: false,
+  }
+
+  render() {
+    const { swapImage } = this.state;
+
+    const {
+      classes, isOpen, onClose, armorData, loadouts, armorToLoadout,
+    } = this.props;
+
+    const { builds } = loadouts;
+
+    const {
+      name, defense, type, resistances, crafting, skills, assets,
+    } = armorData;
+
+    const maleArmor = _.get(assets, "imageMale"),
+      femaleArmor = _.get(assets, "imageFemale");
+
+    const materials = _.get(crafting, "materials");
+
+    const dragonRes = _.get(resistances, "dragon"),
+      waterRes = _.get(resistances, "water"),
+      fireRes = _.get(resistances, "fire"),
+      iceRes = _.get(resistances, "ice"),
+      thunderRes = _.get(resistances, "thunder");
+
+    const baseDef = _.get(defense, "base"),
+      maxDef = _.get(defense, "max"),
+      augmentedDef = _.get(defense, "augmented");
+
+    const data = [
+      {
+        resistance: `Dragon ( ${dragonRes} )`, number: dragonRes,
+      },
+      {
+        resistance: `Water ( ${waterRes} )`, number: waterRes,
+      },
+      {
+        resistance: `Fire ( ${fireRes} )`, number: fireRes,
+      },
+      {
+        resistance: `Ice ( ${iceRes} )`, number: iceRes,
+      },
+      {
+        resistance: `Thunder ( ${thunderRes} )`, number: thunderRes,
+      },
+    ];
+    return (
+      <SlidingModal
+        isOpen={isOpen}
+        onClose={onClose}
+      >
+        <Card
+          className={classes.card}
+          style={{
```

Comment:
im wondering if you can style along with the class, adding separate style is probably gonna make the file bigger